apps/blehci/syscfg.yml

```diff
@@ -16,6 +16,15 @@
 # under the License.
 #
 
+syscfg.defs:
+    CONSOLE_MODE:
+        description: Which console to use
+        value: stub
+        choices:
+            - full
+            - minimal
+            - stub
```

Comment:
I'd add this setting to sys/console instead so we can include sys/console instead of specific package in app and select via syscfg, similar as we did it for hci transport some time ago